docs/import/create_score_asset.js

```diff
@@ -49,7 +39,47 @@ function countDamageAndBuildings(feature, damage, buildings) {
   } else {
     properties = properties.set(damageTag, 0);
   }
-  return ee.Feature(geometry, properties);
+  const snapPop = feature.get(snapPopTag);
+  const totalPop = feature.get(totalPopTag);
+  // unfortunately, just using .contains(null) here throws an error.
+  const snapPercentage = ee.Algorithms.If(
```

Comment:
This works, but might be slightly more straightforward to do `ee.Algorithms.If(snapPop, ee.Algorithms.If(totalPop, ee.Number(snapPop).long().divide(ee.Number(totalPop).long()), null), null)`. Up to you.